thrift-gen/jaeger/tchan-jaeger.go

```diff
@@ -43,10 +43,6 @@ func (c *tchanCollectorClient) SubmitBatches(ctx thrift.Context, batches []*Batc
 	}
 	success, err := c.client.Call(ctx, c.thriftService, "submitBatches", &args, &resp)
 	if err == nil && !success {
-		switch {
```

Comment:
Seems generated deletion looks weird. Why no logging?

Reply:
I can't help it, it's how the recent generator works. But the new code looks correct - propagating the error up, rather than introducing that odd override.

Also, low-level libraries are not supposed to log.